### ISSUE TITLE
[core] provide accessibility user setting to configure toast timeout

### ISSUE THREAD

**Issue by susnux** [nextcloud/server#58387]
> [!TIP]
> ### Help move this idea forward
> * Use the 👍 reaction to show support for this feature.
> * Avoid commenting unless you have relevant information to add; unnecessary comments create noise for subscribers.
> * Subscribe to receive notifications about status changes and new comments.
---

**Is your feature request related to a problem? Please describe.**
Users should have adequate time to complete tasks e.g. reading toast notifications.

**Describe the solution you'd like**
User configurable timeout for toasts.
This could be configured in the accessibility settings.

It must then be used by all toast messages ( nextcloud dialogs library).

**Additional context**
https://www.w3.org/WAI/WCAG21/Understanding/timing-adjustable.html
